go.mod

```diff
@@ -5,12 +5,33 @@ go 1.17
 require (
 	github.com/go-logr/logr v1.2.0
 	github.com/onsi/gomega v1.17.0
-	golang.org/x/crypto v0.0.0-20210921155107-089bfa567519
+	github.com/openshift/api v3.9.0+incompatible
+	github.com/openstack-k8s-operators/keystone-operator v0.0.0-20220516151700-a6967a7f8a1b
+	github.com/openstack-k8s-operators/mariadb-operator v0.0.0-20220516121356-119f8d825a71
```

Comment:
right now used in unit tests, should be changed that we do not have a dependency to operators

Reply:
ugh, yeah this is really bad I think. A cyclical dependency shouldn't happen here

Reply:
I removed the keystone dependency. as discussed on slack we might move the database package to the mariadb operator in a follow up